third_party/yarr/UnicodePatternTables.h

```diff
@@ -1,5 +1,5 @@
 /*
-* Copyright (C) 2017 Apple Inc. All rights reserved.
+* Copyright (C) 2017-2019 Apple Inc. All rights reserved.
```

Comment:
Why did you add `2019` at here?

Reply:
I generated the `UnicodePatternTables.h` with the `generateYarrUnicodePropertyTables.py` from webkit's repository. The script generated the new license header. 

https://github.com/WebKit/webkit/blob/master/Source/JavaScriptCore/yarr/generateYarrUnicodePropertyTables.py